src/pop/c/entities/vbkblock.cpp

```diff
@@ -168,12 +168,11 @@ POP_ENTITY_SERIALIZE_TO_VBK(vbk_block) {
   return res;
 }
 
-POP_ENTITY_DESERIALIZE_FROM_VBK(vbk_block, POP_ENTITY_NAME(config) * config) {
+POP_ENTITY_DESERIALIZE_FROM_VBK(vbk_block) {
   VBK_ASSERT(state);
-  VBK_ASSERT(config);
   VBK_ASSERT(bytes.data);
-  VBK_ASSERT(config->ref);
-  VBK_ASSERT(config->ref->alt);
+
+  auto config = pop_config_new();
```

Comment:
need to dealocate this variable at the end of the function

Reply:
fix 295e151